loom/script/native/lsNativeInterface.cpp

```diff
@@ -204,7 +204,8 @@ void NativeInterface::resolveScriptType(Type *type)
         }
     }
 
-    lmAssert(0, "Unable to resolve script type %s", type->getFullName().c_str());
+    type->setMissing("resolveScriptType");
```

Comment:
Ah, so I overrode this in `lsType` so it works like printf and some call it with more/dynamic information e.g. displaying the missing member name.

So `setMissing` has to be in `MemberInfo` since there's also the `lsr_pushmethodbase` which sets `missing` on a `MethodBase` if that method is missing and `MethodBase` doesn't extend `Type` (obviously). The message there gets lost though. So we could remove the `message` param from the `MemberInfo` definition and add a verbose print in an overridden `MethodBase`.

Does that make any sense? :)